general/clients/installing-kodi.md

```diff
@@ -2,40 +2,41 @@
+1. Download the repository installer [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip).
     * It will be saved as `repository.jellyfin.kodi.zip`
-2. Install the Jellyfin repository
-    * Open Kodi and navigate to "Add-on Browser"
-    * Select "Install from zip file"
+2. Install the Jellyfin repository.
+    * Open Kodi and navigate to "Add-On Browser"
+    * Select "Install from Zip File"
         * If prompted, enter settings and enable "Unknown Sources", then go back
     * Select the newly downloaded file and it will be installed
-3. Install Jellyfin for Kodi
-    * From within Kodi, navigate back to "Add-on Browser"
-    * Select "Install from repository"
+3. Install Jellyfin for Kodi.
+    * From within Kodi, navigate back to "Add-On Browser"
+    * Select "Install from Repository"
     * Choose "Kodi Jellyfin Addons"
     * "Video Add-ons"
     * Select the Jellyfin add-on and choose install
-4. Within a few seconds you should be prompted for your server-details.
-    * If a Jellyfin server is detected on your local network, it will displayed in the popup
-    * If a Jellyfin server is not detected on your local network, select "Manually add server"
-        * Note that if you have a baseurl set, you should append that value to the end of the host field.  ie: `192.168.0.10:8096/jellyfin`
-        * Enter the server name or IP address and the port number (default is 8096)
+4. Within a few seconds you should be prompted for your server details.
+    * If a Jellyfin server is detected on your local network, it will displayed in a dialog
+    * If a Jellyfin server is not detected on your local network, select "Manually Add Server"
+        * Note that if you have a baseurl set, you should append that value to the end of the host field. (`192.168.0.10:8096/jellyfin`)
+        * Enter the server name or IP address and the port number (default value is 8096)
             * Host: `192.168.1.10:8096`
-        * If using SSL and a reverse proxy, enter the full URL scheme in the "Host" field
+        * If using SSL and a reverse proxy, enter the URL in the "Host" field
             * Host: `https://jellyfin.example.com`
     * Select user account and input password
-5. Once you're succesfully authenticated with the Jellyfin server, you'll be asked about your preferences for this device
+5. Once you're succesfully authenticated with the server, you'll be asked about your preferences for this device.
     * Choose your preferences for each options
       * Add-On mode requires no additional configuration
-      * To get Native mode to work, a few things must be done before the files are accessible, so skipping the initial import is necessary, skip the below steps and continue with the **Native Mode Configuration** steps
+      * To get Native mode to work skipping the initial import is necessary, so stop here and skip to **Native Mode Configuration**
```

Comment:
That Native Mode Configuration should probably be an xref link.

Reply:
This is a section in the same file though, so I don't think xref is needed here.

Reply:
Fair enough I guess, although now they are close but they might be several page window sizes away at some point. The xref will point to the local anchor.

Reply:
I'll be doing more style work in future PRs, we can work on proper links then.